content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Monitor Capacitor Applications Using the Browser SDK
+description: Learn how to monitor cross-platform applications built with Capacitor through the browser RUM SDK.
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+## Overview
+
+[Capacitor][1] is an open source native runtime for building Web Native applications that run natively on iOS, Android, and progressive web applications with JavaScript, HTML, and CSS.
+
+You can install and configure the Datadog Browser SDK to start monitoring applications you've built with Capacitor. The configuration provides you with visibility into the JavaScript portion of your application (excludes native application visibility).
```

Comment:
i woudl rpobably clarify what is "(excludes native application visibility)" is, maybe as "(excludes native application visibility e.g. native app startup time, native crashes etc.)"